src/webview/MessageList.js

```diff
@@ -248,7 +247,7 @@ class MessageListInner extends Component<Props> {
 
     // Paranoia^WSecurity: only load `baseUrl`, and only load it once. Any other
     // requests should be handed off to the OS, not loaded inside the WebView.
-    const onShouldStartLoadWithRequest: (event: WebViewNavigation) => boolean = (() => {
+    const onShouldStartLoadWithRequest = (() => {
```

Comment:
```
Rather than go import and use the name the libdef invents for
this callback-argument type, we can just let Flow infer the type.
```
Indeed, yeah. Letting Flow infer here is better than annotating, as I'd done in my PR, for the reasons you mention. 🙂